styles.css

```diff
@@ -81,7 +81,7 @@ body {
 .search-ideas {
   padding-top: 15px;
   padding-bottom: 15px;
-  padding-left: 2px;
+  padding-left: 8px;
```

Comment:
Nice call. It's little details like that that really help improve the user experience.